src/Icon/Icons/ArrowBack.component.tsx

```diff
@@ -0,0 +1,31 @@
+// REACT
+import * as React from 'react';
+import classNames from 'classnames';
+import { IconSVGProps, Scale, StyledIcon } from '../utils';
```

Comment:
This seems to be 1 of 2 components (`ArrowForward` is the other) that doesn't use `DefaultColor` and `DefaultScale`. Is that intentional?

Reply:
+1

Reply:
Good catch. I'll fix that up.